generators/server/templates/src/main/java/package/config/_CacheConfiguration.java

```diff
@@ -277,4 +291,60 @@ public SessionRegistry sessionRegistry() {
     }
     <%_ } _%>
     <%_ } _%>
+
+    <%_ if (hibernateCache == 'infinispan' || clusteredHttpSession == 'infinispan') { _%>
+    private final Logger log = LoggerFactory.getLogger(CacheConfiguration.class);
+
+    /* for customizing session cookie key; default is 'session'
+    @Bean
+    public org.springframework.session.web.http.CookieSerializer cookieSerializer() {
```

Comment:
Yes it does. If I am not wrong, in eclipse u can do cntrl+alt+o, but in Intellij, you have to delete few characters backwards to activate intellisense completion. I just included it so that users can import the write class when they uncomment.